editoast/migrations/2024-03-12-143107_make_budget_nullable/down.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE study ALTER COLUMN budget SET NOT NULL;
+ALTER TABLE project ALTER COLUMN budget SET NOT NULL;
```

Comment:
We should replace all NULL values with `0`. Otherwise, the migration will fail.

Reply:
Done.